waddrmgr/tapscript.go

```diff
@@ -94,6 +101,9 @@ func (t *Tapscript) TaprootKey() (*btcec.PublicKey, error) {
 			t.ControlBlock.InternalKey, t.RootHash,
 		), nil
 
+	case TaprootFullKeyOnly:
+		return t.ControlBlock.InternalKey, nil
```

Comment:
So were sort of overloading the internal key in the control block as the _output_ key?

Reply:
I added an extra `FullOutputKey` field to make this more obvious.